game/src/contrib/utils/components/Debugger.java

```diff
@@ -44,6 +44,7 @@
 public class Debugger {
 
     private static final Logger LOGGER = Logger.getLogger(Debugger.class.getName());
```

Comment:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(Debugger.class.getSimpleName());
```

scheint im logger wohl besser auszusehen